Mage.Sets/src/mage/cards/s/SettleTheWilds.java

```diff
@@ -0,0 +1,75 @@
+
+package mage.cards.s;
+
+import mage.abilities.Ability;
+import mage.abilities.effects.OneShotEffect;
+import mage.abilities.effects.common.SeekCardEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.CardType;
+import mage.constants.ComparisonType;
+import mage.constants.Outcome;
+import mage.constants.Zone;
+import mage.filter.StaticFilters;
+import mage.filter.common.FilterPermanentCard;
+import mage.filter.predicate.mageobject.ManaValuePredicate;
+import mage.game.Game;
+import mage.players.Player;
+
+import java.util.UUID;
+
+/**
+ * @author Sidorovich77
+ */
+public final class SettleTheWilds extends CardImpl {
+
+    public SettleTheWilds(UUID ownerId, CardSetInfo setInfo) {
+        super(ownerId, setInfo, new CardType[]{CardType.SORCERY}, "{2}{G}");
+
+        // Seek a basic land card and put it onto the battlefield tapped.
+        // Then seek a permanent card with mana value equal to the number of lands you control.
```

Comment:
Must use one line per ability for rules text as comment for better text and regexp searching by IDE